### PR TITLE
Claire fixes 

### PR DESCRIPTION
**Pull request by miha-bhaskaran**
# ✨ New in this PR ✨
resolved claire's dev notes fixes

Claire's Notes:
Desktop View
[DONE] check spacing between the navbar, path, and "Wildlife Care Center Maps" title
[DONE] this might be chris's part but the navbar looks like hunter green instead of scary green so maybe someone can check that?
[DONE] the "x" in the top right of the pop up is the color "silver", rn its darker
[DONE] the pop up looks shorter in height and width so check that its length is matched w designs (fixing this might help the pop up hug the edges of the map more closely (22px from the right and bottom edge))
**Mobile View**
[DONE] all elements on the page should line up their left and right most edges (page margin should be 18px) --> i cant tell if the toggle button is too wide or the map is too narrow
[DONE] readjust the dimensions of the mobile pop up --> pls inspect designs
[adding a lot of spacing here makes the pop up bigger - literally tried every possible padding size and div placement so slightly differ than the figma design] check spacing between the "buildings and services" pop up tag and the title text -- these pop ups should live more at the top of the popup
[DONE] check that the tag has enough padding around (padding: 2px 8px) + the text is color "shadow" -- it looks slightly darker rn cant tell if its just the loom recording
[I think that was due to the pop up size - the font style aligns with figma] check that the text under the pop up title is S3, looks like slightly bigger rn? or maybe it could be the effect of the popup being smaller not sure haha

## REQUIRED: Link a short video demo of the feature you have added. 👩‍💻
https://www.loom.com/share/1562ff6b52264fc99c3d79af28acb92a?sid=83b12d2d-43ad-4bb3-8441-a16ab668850a

## Resources 📔

_REQUIREMENTS in this section: Link all relevant online resources and related PRs. Write down any new dependencies or packages and link to their documentation. If you ran into bugs, link any stack overflow or online resources that helped you unblock yourself_

## ❗IF this sprint was marked as FRONTEND on linear, cc Claire Wang

_REMEMBER: once PR is made, also request a PR review Sarah, CC'ing is NOT enough!_
🏚 cc: @sarahhpeng @andreisito

### CODE REVIEW

**Review by sarahhpeng — APPROVED** [review]

File: src/app/siteMapPage/page.tsx [hunk 1]
```diff
@@ -22,51 +25,126 @@ type ModeState = 'tours' | 'exhibits';
...
 function MapPage() {
+
   const [selectedMap, setSelectedMap] = useState(filterButtonContent[0]); // "Virtual Tour Map" by default
   const [mode, setMode] = useState<ModeState>('tours');
+  // const [isWide, setIsWide] = useState(window.innerWidth >= 768);
+  const [isWide, setIsWide] = useState(false);
+
+  useEffect(() => {
+    /**
+     *
```
Comment (jxmoose) [inline src/app/siteMapPage/page.tsx:36]: add comment hehe

File: src/app/siteMapPage/page.tsx [hunk 2]
```diff
@@ -22,51 +25,126 @@ type ModeState = 'tours' | 'exhibits';
...
-    <div className="mb-6  pl-0 pr-0 ">
-      <div className='pt-9 pr-2 pl-2 pb-7'>
+    <div className="mb-6  w-[22.125rem]">
+      {/* <div className='pt-9 pr-2 pl-2 pb-3'>
       <p className="text-night font-lato text-2xl font-semibold">Wildlife Care Center Maps</p>
-      </div>
-      <div className="flex flex-row items-center gap-x-0 pr-1 pl-1 w-full rounded-lg bg-mint-cream  border-mint-cream border-[5px]">
+      </div> */}
+      <div className="flex flex-row items-center pr-0 pl-0 rounded-lg bg-mint-cream  border-mint-cream border-[8px]"
+      style={{ width: '93%', marginLeft: 'auto', marginRight: 'auto' }}>
```
Comment (jxmoose) [inline src/app/siteMapPage/page.tsx:66]: convert this into tailwind unless theres a reason

File: src/app/siteMapPage/page.tsx [hunk 3]
```diff
@@ -22,51 +25,126 @@ type ModeState = 'tours' | 'exhibits';
...
             />
+          
+          ))}
+      </div>
+    </div>
+  );
+  const renderFilterContainerWide = () => (
+    <div className="flex justify-end w-[23.875rem] h-[3.25rem]">
+      <div className="flex flex-row items-center rounded-lg bg-mint-cream  border-mint-cream border-[8px]"
+      style={{ width: '100%', marginLeft: 'auto', marginRight: 'auto' }}>
```
Comment (jxmoose) [inline src/app/siteMapPage/page.tsx:86]: convert tailwind

File: src/app/siteMapPage/page.tsx (hunk 1 above)
Comment (jxmoose) [reply src/app/siteMapPage/page.tsx:36]: should this just be the web resizing @andreisito

File: src/app/siteMapPage/page.tsx [hunk 4]
```diff
@@ -22,51 +25,126 @@ type ModeState = 'tours' | 'exhibits';
...
+    <>
+   
+    
       <NavBar />
+      <div className='justify-center mx-auto w-[70%]'>
+        <div className="pt-0 pl-2 pr-2 bg-ivory">
+
+      <div className="pt-20 pb-6 pl-2 ">
+        <span className="text-scary-forest font-lato text-sm font-normal">
+          Home  <span className='text-night'> / Wildlife Care Center Maps </span>
```
Comment (jxmoose) [inline src/app/siteMapPage/page.tsx:114]: should use text tag instead of span

File: src/components/userComponents/SiteMap/ExhibitPreviewCard.tsx [hunk 5]
```diff
@@ -37,7 +33,27 @@ function ExhibitPreviewCard({
   const [name1, setname1] = useState<string>('');
 
   const { id, description, coordinates, category } = tour;
-  // name, for tour title for exhibit
+  const [width, setWidth] = useState('20.06rem');
+  const [height, setHeight] = useState('7.687rem');
+
+  useEffect(() => {
```
Comment (jxmoose) [inline src/components/userComponents/SiteMap/ExhibitPreviewCard.tsx:39]: change to andreis web conditional?

File: src/components/userComponents/SiteMap/TourPreviewCard.tsx [hunk 6]
```diff
@@ -5,138 +5,91 @@ import Link from 'next/link';
...
+  const [width, setWidth] = useState('20.06rem');
+  const [height, setHeight] = useState('7.687rem');
 
-  // Map Context
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const map = useMapEvents({
-    click: e => {
-      if (!e.latlng.equals(coordinates as LatLngExpression)) {
-        handleClose();
+  useEffect(() => {
```
Comment (jxmoose) [inline src/components/userComponents/SiteMap/TourPreviewCard.tsx:26]: andrei web mayhaps

File: src/components/userComponents/navBar/navBar.tsx [hunk 7]
```diff
@@ -24,27 +24,29 @@ export default function NavBar() {
...
-        >
-          <div className="flex justify-center items-center w-full h-full">
-            {showMenu ? <CloseMenu /> : <HamburgerMenu />}
-          </div>
-        </button>
-      </div>
-      <div className="bg-hunterGreen w-full h-[0.375rem] absolute bottom-0" />
+    <nav
+      className="bg-hunterGreen p-4 flex items-center justify-between z-[9999]"
+      style={{ padding: '1rem', position: 'sticky', top: 0 }}
```
Comment (jxmoose) [inline src/components/userComponents/navBar/navBar.tsx:29]: convert to tailwind

**Review by jxmoose — COMMENTED** [review]
looks good just some style changes ig lol slay queen :3

**Pull request merged**